src/routes.js

```diff
@@ -21,7 +21,7 @@ InsightsRoute.propTypes = {
   rootClass: PropTypes.string,
 };
 
-export const Routes = () => {
+export const Routes = (isOrgAdmin) => {
```

Comment:
Wrong consumption of props
```suggestion
export const Routes = ({ isOrgAdmin }) => {
```